package.json

```diff
@@ -83,6 +83,7 @@
     "angular-route": "1.4.7",
     "angular-sanitize": "1.5.7",
     "angular-sortable-view": "0.0.15",
+    "apollo-boost": "^0.1.16",
```

Comment:
Should this be in devDependencies instead?

Reply:
Er... that's weird. Good catch.